ktor-client/ktor-client-core/common/src/io/ktor/client/engine/HttpClientEngineConfig.kt

```diff
@@ -14,6 +14,10 @@ public open class HttpClientEngineConfig {
     /**
      * Specifies network threads count advice.
      */
+    @Deprecated(
+        "The [threadsCount] property is deprecated. The [Dispatchers.IO] is used by default.",
```

Comment:
Is it okay to have a deprecation error in a patch release?

Reply:
Let me make it as warning